src/System.Net.Mail/tests/Functional/SmtpExceptionTest.cs

```diff
@@ -86,24 +86,6 @@ public void TestConstructorWithStringArgument()
         }
 
         [Fact]
-        public void TestConstructorWithSerialization()
-        {
-            string msg = "MESSAGE";
-            Exception inner = new ArgumentException("whatever");
```

Comment:
Question: would `ArgumentException` still be `[Serializable]` and implementing `ISerializable`?

We have tests for [DataContractSerializer serializing `ArgumentException`](https://github.com/dotnet/corefx/blob/178a982df2f210b615e21af7b0920f0fe5425142/src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs#L1454). I wondered if it's still supported.